frontend/src/style/ThemeProvider.tsx

```diff
@@ -141,6 +145,12 @@ const defaultPalette = {
         },
         name: 'trainingPlanTaskComplete',
     }),
+    dullGrey: defaultTheme.palette.augmentColor({
+        color: {
+           main: '#FFFFFF', 
```

Comment:
Why are you defining white as a new color?

Reply:
it was dully grey actually was a typo